include/mod_privacy.hrl

```diff
@@ -18,21 +18,26 @@
 %%%
 %%%----------------------------------------------------------------------
 
--record(privacy, {us,
-		  default = none,
-		  lists = []}).
-
--record(listitem, {type = none,
-		   value = none,
-		   action,
-		   order,
-		   match_all = false,
-		   match_iq = false,
-		   match_message = false,
-		   match_presence_in = false,
-		   match_presence_out = false
-		  }).
-
--record(userlist, {name = none, list = [], needdb = false }).
+-record(privacy, {
+          us             :: jid:simple_bare_jid(),
+          default = none :: mod_privacy:list_name(),
+          lists   = []   :: [{mod_privacy:list_name(), mod_privacy:list_item()}]
+         }).
```

Comment:
Minor: please do not add these extra spaces, the whole block would need reformatting when the longest line is changed.